src/components/timeline/form/TimelineEditorPanel.svelte

```diff
@@ -978,6 +960,141 @@
             </Input>
           </form>
         </fieldset>
+      {:else if rowHasExternalEventLayer}
```

Comment:
We should show both Activity Event Options and External Event Options so that you can control both of these layers if you have them on the same row. For example a user might want to plot both events and activities in compact mode.

https://github.com/user-attachments/assets/db31e8b7-99a5-4245-a7ad-3b13420f8945